app/renderer/components/Inspector/Inspector.js

```diff
@@ -40,6 +75,17 @@ export default class Inspector extends Component {
     this.props.bindAppium();
     this.props.applyClientMethod({methodName: 'source'});
     this.props.getSavedActionFramework();
+    window.addEventListener('resize', this.updateSourceTreeWidth);
+  }
+
+  componentDidUpdate () {
+    const {screenshot} = this.props;
+    // only update when the screenshot changed, not for any other kind of
+    // update
+    if (screenshot !== this.lastScreenshot) {
```

Comment:
should we also check here if `screenshot` is defined?

Reply:
it shouldn't matter. if it is not defined, our `(!img)` check will bail us out later on.